integration_tests/integration/createLicence.cy.ts

```diff
@@ -61,6 +61,7 @@ context('Create a licence', () => {
         .nextCondition()
         .checkBoxes()
         .nextCondition(false) // aria-expanded attribute causes issues with Axe
+        .selectRadio('One curfew')
```

Comment:
Would be good to have better test coverage for multiple conditions... lack of test coverage really hampers ability to maintain code mid to long term. Restricts ability to safely refactor code